concepts/language-model.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model tries to predict the next word given the already produced history.
+For example, given the text span `the man with` a language model's output could be `a: 0.4, the: 0.3, my: 0.1, his: 0.05, ...`.
```

Comment:
I'd avoid putting this in prose, and try something more like:

> Example
> > **Input** sequence
> > `The man is riding a _____`
> > **Output** tokens and probabilities
> > `bike`: 0.4, `motorcycle`: 0.3, `scooter`: 0.1, `horse`: 0.05, ...

Reply:
Reformated.